examples/default/domain/aggregate.go

```diff
@@ -75,7 +75,7 @@ func (i *InvitationAggregate) HandleCommand(command eh.Command) error {
 			return nil
 		}
 
-		i.StoreEvent(&InviteAccepted{i.AggregateID()})
+		i.StoreEvent(&InviteAccepted{i.AggregateID().(eh.UUID)})
```

Comment:
I hope the other changes to the project does not cause too much trouble rebasing for you.